clarity/evaluator/msbg/msbg_utils.py

```diff
@@ -525,7 +524,7 @@ def pad(signal: ndarray, length: int) -> ndarray:
     )
 
 
-def read_signal(
+def read_signal_x(
```

Comment:
Could this function not be removed too? Code looks very similar, at the minimum could have the file read by `utils.file_io.read_signal()` and then resample `if wave_file.samplerate != MSBG_FS:`.

Reply:
Yes! Thanks. The rename to read_signal_x was just a step I made during porting to quickly see where it was being used :-). I then forgot to delete the function.